android/app/src/main/java/org/openbot/env/NetworkServiceConnection.java

```diff
@@ -149,12 +147,7 @@ public void onServiceLost(NsdServiceInfo service) {
           ((Activity) context)
               .runOnUiThread(
                   () -> {
-                    try {
```

Comment:
Why is this no longer needed?

Reply:
I moved the try/catch to the function `ControllerToBotEventBus.emitEvent()` to make the calls cleaner, instead of checking with every call.

Reply:
I moved the try/catch to the function `ControllerToBotEventBus.emitEvent()` to make the calls cleaner, instead of checking with every call.